pkg/pgmodel/sql_reader.go

```diff
@@ -439,3 +361,142 @@ func (errorSeriesSet) Next() bool                   { return false }
 func (errorSeriesSet) At() storage.Series           { return nil }
 func (e errorSeriesSet) Err() error                 { return e.err }
 func (e errorSeriesSet) Warnings() storage.Warnings { return nil }
+
+// LabelNames implements the Querier interface. It returns all distinct
+// label names available in the database.
```

Comment:
I do feel mentioning `labels table` is more useful for the contributors that `in the database`. Then, it sounds in lines of prometheus tsdb design, but actually being a table here.

Reply:
I don't think that specific detail is relevant at this level of code documentation since its a implementation detail.